### PR TITLE
idz code review - add support for custom syslib

### PR DESCRIPTION
**Pull request by dennis-behm**
Please see an enhancement to provide a custom SYSLIB concatenation through the properties file. See #71 

Couple of additional comments:
- I switched from standard JAVA Properties to DBB BuildProperties.
- I had to rename the `properties` variable to `props`. I have no idea, but when using `properties` then `properties.workDir` turned `null` within an each loop to extract the DD Outputs, even that @Field was defined ... I have not found that it is a reserved word or anything.

### CODE REVIEW

File: Pipeline/RunIDZCodeReview/RunCodeReview.groovy [hunk 1]
```diff
@@ -173,20 +173,22 @@ def createCodeReviewExec(String jobcard, String ruleFile, String customRuleFile,
 //*
 //AKGCREV EXEC PROC=AKGCR
 """
-	steplib.eachWithIndex {it, index ->
+	jcl+="//SYSLIB   DD DUMMY \n"
+	// add identified syslib
+	syslib.eachWithIndex {it, index ->
 		if (index == 0 ) jcl+="//SYSLIB   DD DISP=SHR,DSN=${it} \n"
```
Comment (jbyibm) [inline Pipeline/RunIDZCodeReview/RunCodeReview.groovy:179]: We can have 2 SYSLIB   in a JCL ? The second will override the first ?
Maybe we can add //SYSLIB   DD DUMMY only if syslib is empty ?

File: Pipeline/RunIDZCodeReview/codereview.properties [hunk 2]
```diff
@@ -26,4 +26,9 @@ codereview_includedIncludeFiles=**/*.cpy
...
 ## Specify the L=COBOL parameter for the Code Review List mapping
 codereview_languageMapping=COBOL :: **/*.cbl
-codereview_languageMapping=PLI :: **/*.pli
\ No newline at end of file
+codereview_languageMapping=PLI :: **/*.pli
+
+#
+## Additional datasets added to the SYSLIB concatenation for the code review application
+## codereview_syslib=JENKINS.EXTERNAL.COPY
+codereview_syslib=
```
Comment (jbyibm) [inline Pipeline/RunIDZCodeReview/codereview.properties:34]: Can you explicitly say that it is a comma separated list of datasets.

**Review by jbyibm — CHANGES_REQUESTED** [review]
See my comments

**Comment by dennis-behm** [comment]
@jbyibm I have updated and included your feedback.

File: Pipeline/RunIDZCodeReview/RunCodeReview.groovy (hunk 1 above)
Comment (dennis-behm) [reply Pipeline/RunIDZCodeReview/RunCodeReview.groovy]: See update

File: Pipeline/RunIDZCodeReview/codereview.properties (hunk 2 above)
Comment (dennis-behm) [reply Pipeline/RunIDZCodeReview/codereview.properties]: See update

**Review by jbyibm — APPROVED** [review]
look good

**Pull request merged**